src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -81,12 +81,24 @@ public void setFabric8(Config fabric8) {
 		 */
 		private String memory;
 
+		/**
+		 * Container GPU vendor limit.
```

Comment:
"Container GPU vendor name for limit" - this content is extracted and auto populated into UI components, and seems a little more descriptive